appengine-skeleton-archetype/src/main/resources/archetype-resources/src/main/java/HelloAppEngine.java

```diff
@@ -0,0 +1,37 @@
+#set( $symbol_pound = '#' )
+#set( $symbol_dollar = '$' )
+#set( $symbol_escape = '\' )
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ${package};
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class HelloAppEngine extends HttpServlet {
+
+  public void doGet(HttpServletRequest request, HttpServletResponse response) 
+      throws IOException {
+      
+    response.setContentType("text/plain");
+    response.getWriter().println("Hello App Engine!");
+
+  }
+}
```

Comment:
needs @ Override
In general, I would check that all 3 IDEs eclipse, intellij, Netbeans do not emit any warnings or recommendations on all xml/jsp/css/java code emitted by a project template.
All code should also pass the Google style guide.